x-pack/plugins/graph/public/components/settings/blocklist_form.tsx

```diff
@@ -41,7 +42,9 @@ export function BlocklistForm({
             <FormattedMessage
               id="xpack.graph.blocklist.noEntriesDescription"
               defaultMessage="You don't have any blocked terms. Select vertices and click {stopSign} in the control panel on the right to block them. Documents that match blocked terms are no longer explored and relationships to them are hidden."
-              values={{ stopSign: <span className="kuiIcon fa-ban" /> }}
+              values={{
+                stopSign: <EuiIcon type="filterIgnore" />,
```

Comment:
Same as previous question. Would it be better to use `eyeClosed` here instead and change all references of "block" to "hide"?

Reply:
Yeah, exactly. Something like "Hidden list" or "Hidden vertices".

Reply:
I've created a follow up issue to track this: https://github.com/elastic/kibana/issues/162297